varya/default-style/style.css

```diff
@@ -27,31 +27,154 @@ a {
 	border-bottom: 1px solid var(--global--color-secondary);
 }
 
+a:focus {
+    color: var(--global--color-secondary);
+}
+
+a:focus {
+    outline: 1px dotted var(--global--color-secondary);
+    border-bottom: none;
+}
+
+a:active,
+a:hover {
+    border-bottom-width: 2px;
```

Comment:
Making the border wider here will make the actual box taller for `block` & `inline-block` items: 

![jump](https://user-images.githubusercontent.com/1202812/77787718-6566c400-7036-11ea-9a92-022e527261f7.gif)

To prevent that jump, can we add an additional 1px bottom box-shadow instead?

Reply:
Yep, good call.